### ISSUE TITLE
No focus state on card-buttons makes keyboard navigation difficult

### ISSUE THREAD

**Issue by humancompanion** [USAJOBS/design-system#148]
The lack of a focus state on the card-button component makes keyboard navigation difficult, if not downright impossible. Reported by @wakaye and @margaretdumas. This is a problem where the component has been used: in Profile for Hiring Paths and in Sign for ATP.

**Comment by humancompanion** [comment]
@margaretdumas notes that Firefox shows the focus oddly:

![image 1](https://user-images.githubusercontent.com/352987/47309840-74587400-d603-11e8-8105-f84b30d269a4.png)

IE does not show the focus.

@starrchen - Can you take this one and try to determine the following:

* [ ] Fix the focus display for Firefox.
* [ ] Determine why the focus does not display in IE and recommend a fix

Thanks.

**Comment by dreeeeewwwww** [comment]
<img width="303" alt="screen shot 2018-10-23 at 4 43 26 pm" src="https://user-images.githubusercontent.com/13023132/47389505-ee135f00-d6e2-11e8-805a-2d2d616ae006.png">

Looks like absolute positioning that conceals the radio/checkbox is what is throwing it off.

**Comment by dreeeeewwwww** [comment]
- [ ] Change negative left position to `opacity: 0;`
- [ ] Test in FF Mac
- [ ] Test in FF Win
- [ ] Test in Chrome Mac
- [ ] Test in Chrome Win
- [ ] Set up Windows VM to test locally on Edge & IE
